config/common.php

```diff
@@ -5,7 +5,7 @@
 use Psr\Container\ContainerInterface;
 use Yiisoft\Aliases\Aliases;
 
-/* @var array $params */
+/** @var array $params */
```

Comment:
It is vardoc, not phpdoc. The change isn't necessary.

Reply:
done.